src/main/java/com/ibm/as400/access/AS400.java

```diff
@@ -1128,10 +1213,23 @@ public static void addPasswordCacheEntry(String systemName, String userId, Strin
      **/
     public static void addPasswordCacheEntry(String systemName, String userId, char[] password) throws AS400SecurityException, IOException
     {
-        if (Trace.traceOn_) Trace.log(Trace.DIAGNOSTIC, "Adding password cache entry, system name: '" + systemName + "' user ID: '" + userId + "'");
```

Comment:
Why were these trace points removed?

Reply:
because all these methods go through one method that produces a trace, thus no reason to duplicate.  This method and its ilk go through 

    public static void addPasswordCacheEntry(String systemName, String userId, char[] password, String proxyServer, boolean useSSL) throws AS400SecurityException, IOException
    {
        if (Trace.traceOn_) Trace.log(Trace.DIAGNOSTIC, "Adding password cache entry, system name: '" + systemName + "' user ID: '" + userId + "' proxy server: '" + proxyServer + "'");
        addPasswordCacheEntry((useSSL) ? new SecureAS400(systemName, userId, password, proxyServer) : new AS400(systemName, userId, password, proxyServer));
    }